src/xenia/kernel/util/object_table.cc

```diff
@@ -251,7 +250,7 @@ ObjectTable::ObjectTableEntry* ObjectTable::LookupTable(X_HANDLE handle) {
   auto global_lock = global_critical_region_.Acquire();
 
   // Lower 2 bits are ignored.
-  uint32_t slot = handle >> 2;
+  uint32_t slot = (handle - XThread::kHandleBase) >> 2;
```

Comment:
Can `(handle - XThread::kHandleBase) >> 2` be put in an inline (or even constexpr, since the implementation allows) function like ObjectTable::GetSlot?